README.md

```diff
@@ -176,3 +176,6 @@ Lints all packages using [ESLint](https://eslint.org/).
 ### `update`
 
 Interactively updates all dependencies to their latest versions, respecting ranges specified in `package.json`.
+
+## Contribute to Ottehr!
+We love it when you contribute to Ottehr! By submitting to this project, you agree to adopt the [Developer Certificate of Origin (DCO)](https://developercertificate.org/) for your contributions.
```

Comment:
Not sure if this is a template, but I believe using verbs like contribute, commit, collaborate and etc would be better instead of submit

Reply:
that is related to https://github.com/masslight/ottehr/pull/265, the wording was provided by daniel